src/placeos-rest-api/controllers/root.cr

```diff
@@ -102,40 +120,63 @@ module PlaceOS::Api
     # For MQTT JWT access: https://github.com/iegomez/mosquitto-go-auth#remote-mode
     # jwt_response_mode: status, jwt_params_mode: form
     post "/mqtt_user", :mqtt_user do
-      mqtt_parse_token
       head :ok
     end
 
+    @[Flags]
+    enum MqttAcl
+      Read      = 0x01
+      Write     = 0x02
+      Subscribe = 0x04
+      Deny      = 0x11
```

Comment:
Not sure this needs to be a flags, not sure where deny came from either?

The values mosquitto pass are:
1 is read, 2 is write, 3 is readwrite, 4 is subscribe

Reply:
Yep, they're flags. https://github.com/iegomez/mosquitto-go-auth/blob/master/backends/constants/constants.go

Reply:
Read | Write => 0x01 | 0x02 => 0x03

Reply:
makes sense, although probably worth adding a comment explaining the logic there.
man, you went deep

Reply:
and add that link to the source code in the comment too